exchange/exchange_test.go

```diff
@@ -2585,3 +2629,15 @@ type panicingAdapter struct{}
 func (panicingAdapter) requestBid(ctx context.Context, request *openrtb.BidRequest, name openrtb_ext.BidderName, bidAdjustment float64, conversions currencies.Conversions, reqInfo *adapters.ExtraRequestInfo) (posb *pbsOrtbSeatBid, errs []error) {
 	panic("Panic! Panic! The world is ending!")
 }
+
+func blankAdapterConfig(bidderList []openrtb_ext.BidderName) map[string]config.Adapter {
```

Comment:
Now that the exchange accepts a list of bidders directly, we can likely redesign these tests to use fake bidders. That's beyond the scope of this PR.. it's big enough as is.